homeassistant/components/mqtt/lawn_mower.py

```diff
@@ -199,7 +199,7 @@ async def async_start_mowing(self) -> None:
 
     async def async_dock(self) -> None:
         """Dock the mower."""
-        await self._async_operate("dock", LawnMowerActivity.DOCKED)
+        await self._async_operate("dock", LawnMowerActivity.RETURNING)
```

Comment:
I don't think we should change this. It would mean that the entity would never be docked if the entity only changes state due to service calls and doesn't have a state value template.

Reply:
Ok, I'll change that back.